src/util/web3.js

```diff
@@ -147,6 +148,10 @@ export async function getGasPrice(
   networkType,
   { mainnet: { safeMinimum = '3', disableEstimate } = {} } = {}
 ) {
+  if (getNetworkConfig(networkType).settings.disableEstimateGas) {
+    return
+  }
+
   if (!isOnEthMainnet(networkType)) {
```

Comment:
should we clean up all of the hardcoding of gas estimate below and use the flag `disableEstimateGas` instead of having another `disableEstimate` variable?

Reply:
makes sense... updating it